packages/sdk-client-v3/src/types/types.d.ts

```diff
@@ -279,7 +279,7 @@ export type IClientOptions = {
   method: MethodType;
   headers: Record<string, any>
   credentialsMode?: CredentialsMode;
-  body?: string | Buffer
+  body?: Record<string, any> | string | Buffer
```

Comment:
Can we have a consistent definition for the body object type, either we stick with `Record<string, any>` or we go with `{ [p: string]: any }`